ci/launch.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -e -u -x -o pipefail
+
+cd "$(dirname "$0")/../"
+
+source ".shared-ci/scripts/profiling.sh"
+source ".shared-ci/scripts/pinned-tools.sh"
+
+# Download the artifacts and reconstruct the build/assemblies folder.
+buildkite-agent artifact download "build\assembly\**\*" .
+
+uploadAssembly "${PREFIX}" "${PROJECT_NAME}"
+
+markStartOfBlock "Launching deployments"
+
+dotnet run -p workers/unity/Packages/com.improbable.gdk.deploymentlauncher/.DeploymentLauncher/DeploymentLauncher.csproj -- \
+    create "${PROJECT_NAME}" "${ASSEMBLY_NAME}" \
+    "${ASSEMBLY_NAME}" cloud_launch_large.json snapshots/cloud.snapshot \
+    "${ASSEMBLY_NAME}_sim_players" cloud_launch_large_sim_players.json
+
+if [[ -n "${BUILDKITE-}" ]]; then
```

Comment:
If we never run this on TC its probably safe to remove this.. unless we except users to run this script?

Reply:
Nope. Users can use this as an idea of how to use the deployment launcher through scripts if they want, but we shouldn't expect users to run any of our ci/ scripts